docs/CHANGELOG.asciidoc

```diff
@@ -50,6 +50,11 @@ SIEM::
 * When you deactivate an action within a rule, the rule fails {issue}64870[#64870]
 * When you add a timestamp to the PagerDuty action, the rule fails {issue}64871[#64871]
 
+Configuration::
+* The `server.customResponseHeaders` option will prevent {kib} to start if any headers are set using other types than string.
+You can convert your boolean or number headers to strings to fix this. (`my-header: "true"` instead of `my-header: true`).
```

Comment:
```suggestion
To fix this, convert your boolean and number headers to strings.  For example, use `my-header: "true"` instead of `my-header: true`.
```